goodfirstissue/handler.go

```diff
@@ -52,6 +61,17 @@ func Handle(w http.ResponseWriter, r *http.Request) {
 	}
 	logrus.Tracef("%s", string(body))
 
+	//per https://developer.github.com/webhooks/#events integration_installation and integration_installation_repositories events are
+	// deprecated and replaced by installation and installation_repositories. But github is still sending integration_installation and
+	// integration_installation_repositories and go-github don't understand these.
+
+	switch t {
```

Comment:
What about removal events? (are they correctly ignored?)

This code should be a function and tested 👍